edit/buildozer.go

```diff
@@ -1076,6 +1076,8 @@ func appendCommands(opts *Options, commandMap map[string][]commandsForTarget, ar
 		for _, buildFileName := range BuildFileNames {
 			if strings.HasSuffix(target, filepath.FromSlash("/"+buildFileName)) {
 				target = strings.TrimSuffix(target, filepath.FromSlash("/"+buildFileName)) + ":__pkg__"
+			} else if strings.HasSuffix(target, "/"+buildFileName) {
+				target = strings.TrimSuffix(target, "/"+buildFileName) + ":__pkg__"
```

Comment:
This code now replaces the trailing "/BUILD" or "\BUILD" (but in non-windows environment only "/BUILD") with ":__pkg__", leaving all other slashes as they are. That doesn't look correct to me, the slashes should either always be forward if they represent targets or system specific if they represent files. The `target` variable is later being interpreted as a modified file name but sometimes as a label (that should probably be fixed).

I think it's better to replace forward slashes in `target` with system-specific, and have just one call to `strings.HasSuffix`, that should fix your problem, make the behavior more predictable (`target` will always have system-specific slashes), and also make the function slightly more readable.

Reply:
I gave it a try, let me know if I got it right. I tried to do:

> I think it's better to replace forward slashes in target with system-specific

but I think this goes against:

> the slashes should either always be forward if they represent targets 

Because now even when `target` is representing a target we have `\` on Windows.